<launch>

  <node name="dronemap_heartbeat_operator" pkg="dronemap_control" type="heartbeat_operator_node.py"  output="screen">
  </node>
  
</launch>